### ISSUE TITLE
feature request

### ISSUE THREAD

**Issue by am0o0** [nullt3r/udpx#2]
Thanks for the awesome updx tool 
Can you please tell me How Can I add a probe by myself, please?
I want to add HTTP3/QUIC probes that I think are based on the UDP.

**Comment by nullt3r** [comment]
Good idea, I will look into it. Documentation is still work in progress, but there is [this](https://github.com/nullt3r/udpx#how-to-add-your-own-probe) little section. I am thinking about adding YML templates.

**Comment by nullt3r** [comment]
The poor's man solution is to capture the data via netcat, e. g. `nc -ul PORT > /tmp/out.pkt` and then convert the packet to hex data `cat /tmp/out.pkt | hexdump -e '512/1 "%02X"'`. Just make sure to share it so we can add it to the next release.

**Comment by nullt3r** [comment]
Yep, I realised there is TLS 1.3 by design in QUICK, so thats expected.

**Comment by nullt3r** [comment]
This is why, e. g.  Wireguard and such is not included in probes. I will need to make actual module for TLS. If you have any idea, let me know.
